src/dune_rules/dune_file.ml

```diff
@@ -460,7 +461,11 @@ module Mode_conf = struct
               in
               (mode, Kind.Requested loc))
             ~eq:(fun (a, _) (b, _) -> equal a b)
-            ~standard:[ (Ocaml Best, Kind.Requested stanza_loc) ]
+            ~standard:
```

Comment:
I'd add a `Mode_conf.Set.Default.to_list` or something, and we'd be synced by default.

Reply:
Fixed in https://github.com/ocaml/dune/pull/7467/commits/b21c4a3710b86a8458a0e88c25cc4c455a467245.